pkg/utils/utils.go

```diff
@@ -36,18 +39,10 @@ func DetermineHostByteOrder() binary.ByteOrder {
 }
 
 const (
-	systemProcessName      string = "system_processes"
-	systemProcessNamespace string = "system"
+	SystemProcessName      string = "system_processes"
+	SystemProcessNamespace string = "system"
 )
 
-func GetSystemProcessName() string {
```

Comment:
lets keep the access function to protect the variable from being changed outside of the pkg scope.

Reply:
> lets keep the access function to protect the variable from being changed outside of the pkg scope.

do we have this kind of case?
as you can see I removed this Get function and meanwhile make the const able to be access from other packages.
in my point of view, is there any case to modify a const?

Reply:
right, it is already a const, no mutable possible outside the scope.